webapp/ordering/models.py

```diff
@@ -234,6 +240,33 @@ def send_reminder_mails(self):
                 mail_template, user=user, order_round=self)
             mail_user(user, *rendered_template_vars)
 
+    def get_rides(self):
+        from transport.models import Ride
```

Comment:
Inline import - is this because of a circular dependency?

Reply:
Yes indeed. This seemed like a simple solution, especially since I only needed Ride there.

Reply:
By retrieving the rides from the related field I didn't need this anymore.